EventFilter/CTPPSRawToDigi/python/ctppsRawToDigi_cff.py

```diff
@@ -0,0 +1,66 @@
+import FWCore.ParameterSet.Config as cms
+
+# trigger data
+from EventFilter.CTPPSRawToDigi.totemTriggerRawToDigi_cfi import *
```

Comment:
Is `import *` actually necessary? It would be better to explicitly name the modules you need to import.